omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -218,4 +218,12 @@
 		}
 		return appList;
 	}
+
+	@RequestMapping(value = "/rest/owa/allowAdmin", method = RequestMethod.GET)
+	@ResponseBody
+	public boolean allowWebAdmin() {
+		boolean allowWebAdminPrivileges;
+		allowWebAdminPrivileges = ModuleUtil.allowAdmin();
```

Comment:
It is not a good practice to declare a variable that you are not using.

Reply:
I  am assigning the value of the properties to the variable and then returning it.
So the variable is not unused

Reply:
Can't you return without first assigning to a variable? That is what i mean by saying that this variable is unused (useless)

Reply:
okay, i get the point let me fix it.